vou/person.py

```diff
@@ -49,6 +49,12 @@ def __init__(
         self.post_OD_use_pause = None
         self.last_dose_increase = 0
 
+        drug_params = load_json("vou/drug_params.json")
```

Comment:
I suggest adding a single `self.drug_params` rather than pulling out individual parameters as attributes here. That would be more concise and would also avoid additional changes if we want to add/use other parameters.

If you want to avoid repetitive indexing, you can always do something like this at the top of a function. I.e., in `will_increase_dose`, you could add
```
source_probability_primary = self.drug_params['source_probability']['initial_doctor_increase']
```
etc.

Finally, you need to import `load_json` to avoid an import error.